<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>753</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
    <fontweight>Medium</fontweight>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #a2ff94
}

//#a2ff94</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>1221</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Ключ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="keyEdit">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font-size: 15px;
	border-radius: 10px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 10px;
    transition: .1s linear;
    user-select: none;
    line-height: 1.5;
    cursor: pointer;
	background: #189404;
    border: 1px solid #147006;
    font-size: 15px;
    letter-spacing: 1px;
	color: #fff;
}

QPushButton:hover {
    background-color: #157a05;
}</string>
       </property>
       <property name="text">
        <string>Считать из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 10px;
    transition: .1s linear;
    user-select: none;
    line-height: 1.5;
    cursor: pointer;
	background: #189404;
    border: 1px solid #147006;
    font-size: 15px;
    letter-spacing: 1px;
	color: #fff;
}

QPushButton:hover {
    background-color: #157a05;
}</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 10px;
    transition: .1s linear;
    user-select: none;
    line-height: 1.5;
    cursor: pointer;
	background: #189404;
    border: 1px solid #147006;
    font-size: 15px;
    letter-spacing: 1px;
	color: #fff;
}

QPushButton:hover {
    background-color: #157a05;
}</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TreeWidget" name="treeWidget" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget {
	background-color: 3657ff;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QWidget</extends>
   <header>treewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
